public/constants.js

```diff
@@ -1,3 +1,40 @@
+import heroImage from "../public/assets/heroImage.webp";
```

Comment:
The location of this file struck me as odd at first. Is this code going to be executed by the client, or is it just consumed during the build process? If the latter, I am not sure if it belongs in the `public` directory.

Reply:
What do you mean by consumed during the build process?

Reply:
By that, I mean that it is compiled into other files, as opposed to being served to the client.

Reply:
Oh yeah this _is_ served to the client.